src/Humanizer/CollectionHumanizeExtensions.cs

```diff
@@ -15,46 +15,46 @@ public static class CollectionHumanizeExtensions
         /// using the default separator for the current culture.
         /// </summary>
         /// <returns></returns>
-        public static string Humanize<T>(this IEnumerable<T> collection)
+        public static string Humanize<T>(this IEnumerable<T> collection, StringJoinOptions options = StringJoinOptions.Default)
```

Comment:
_**TL;DR:** addingition of overload method is non breaking change; addition of a default argument - is._

It's about binary compatibility, if you add overload and do a dll-drop upgrade - everything breaks. You'll think - why would someone do a dll-drop update? To answer this, let's imagine the situation: 

1. Your project depends on Humanizer and other nuget package "A".
2. "A" depends on Humanizer as well.
3. You update the Humanizer, but "A" still references the older version. So, everything breaks.

We are using SemVer. Addition of a new method is a backward compatible change, so we can increase only minor version part. Removal of a method - is not a backward compatible change, so we forced to increase a major version.